### ISSUE TITLE
`getrawtransaction` fixes and updates

### ISSUE THREAD

**Issue by mpguerra** [ZcashFoundation/zebra#8945]
## Motivation

This method is required for both mining and block explorer usecases. There are a number of outstanding fixes and updates that are necessary in order for this RPC method to match its zcashd counterpart which are listed as subissues.

**Comment by mpguerra** [comment]
We're not quite done here

**Comment by mpguerra** [comment]
Is there anything else to be done here?

**Comment by mpguerra** [comment]
I think we are done here

**Comment by dismad** [comment]
If its not too much trouble, .size and .time would be great! Thanks for the work team \o/

**Comment by mpguerra** [comment]
I think we are done here for now

**Comment by pacu** [comment]
Can we reopen this to add missing fields of verbosity 1?

At least txid as the most pressing one


see 
https://github.com/zcash/zcash-explorer/blob/70da6eeb810bcb45551c31c466587510ceaad57a/lib/zcash_explorer_web/controllers/transaction_controller.ex#L4-L18

**Comment by conradoplg** [comment]
> Can we reopen this to add missing fields of verbosity 1?
> 
> At least txid as the most pressing one
> 
> see https://github.com/zcash/zcash-explorer/blob/70da6eeb810bcb45551c31c466587510ceaad57a/lib/zcash_explorer_web/controllers/transaction_controller.ex#L4-L18

Which ones exactly?

We can add `txid`, though that's really trivial for clients to know (since they need it to make the request...), but I guess is also trivial for us to add.

Do you have a reference to where the transaction is parsed? i.e. `Zcashex.Transaction.from_map` and `Poison.encode!` (btw I don't really get what `get_raw_transaction` does there...)

**Comment by pacu** [comment]
> Do you have a reference to where the transaction is parsed? i.e. `Zcashex.Transaction.from_map` and `Poison.encode!` (btw I don't really get what `get_raw_transaction` does there...)

yes here it's map against all the fields but I bet not all of them are used

https://github.com/zcash/zcashex/blob/main/lib/zcashex/transaction.ex

**Comment by pacu** [comment]
> We can add `txid`, though that's really trivial for clients to know (since they need it to make the request...), but I guess is also trivial for us to add.

yes, but. block explorer UIs may not work that way. at least Zcash's doesn't 🥲

also we need this for `getblock` verbosity 2 which it is used by the explorer too


If these are easy to do for a novice in the Zebra code, I can happily start doing it with guidance and review of the Zebra team when needed

**Comment by mpguerra** [comment]
@pacu could you please create additional issues for each additional field that is needed? Or is it just txid?  

That way we can triage and schedule accordingly. Thank you! :)

**Comment by pacu** [comment]
Excellent! I'll create issues and we can work them out together 💪
